src/backend/memory.rs

```diff
@@ -4,7 +4,7 @@ use alloc::vec::Vec;
 use primitive_types::{H160, H256, U256};
 
 /// Vivinity value of a memory backend.
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, Default)]
```

Comment:
```suggestion
#[derive(Clone, Debug, Eq, PartialEq)]
```

I'd prefer we not implement `Default`. It's meaningless in production. If you use this in tests, it wouldn't take much time to just write down all the values.

Reply:
If I want to put `MemoryVicinity` in a struct that i'd want to implement Default for, the lack of a `Default` implementation for it means I need to also manually implement it for my struct. 

I am OK with doing that if you're strongly against it, but it's a pain and if I can avoid it with a simple derive Default here I'd highly prefer that.

Reply:
Can you show me the actual code you're using this?